iwyu_output.cc

```diff
@@ -605,6 +636,28 @@ void IwyuFileInfo::ReportForwardDeclareUse(SourceLocation use_loc,
   LogSymbolUse("Marked fwd-decl use of decl", symbol_uses_.back());
 }
 
+void IwyuFileInfo::ReportUsingDeclUse(SourceLocation use_loc,
+                                      const UsingDecl* using_decl,
+                                      const NamedDecl* target_decl,
+                                      bool in_cxx_method_body,
+                                      const char* comment) {  
+  for(auto&& saved_decl : using_decl_lines_) {
```

Comment:
I'd prefer spelling out the type here, instead of using `auto` -- I tend to use the latter only when the type is named on the right-hand side of the expression.